lib/tasks/auto_annotate_models.rake

```diff
@@ -0,0 +1,59 @@
+# NOTE: only doing this in development as some production environments (Heroku)
```

Comment:
@zarina86 why added this file?

Reply:
I faced an issue with auto annotations of the model. After bundle, my model didn't annotate automatically. So I run  ```bundle exec rails g annotate:install ```   in terminal